src/pages/Main/Main.css

```diff
@@ -5,20 +5,22 @@ body {
 }
 
 .pageContainer {
-    width: 100%;
-    height: 100%;
+    width: 100vw;
+    height: 100vh;
+    height: calc(var(--vh, 1vh) * 100);;
     position: absolute;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
 .mainContainer {
-    width: 400px;
-    height: 800px;
+    width: 100vw;
+    height: 100vh;
+    height: calc(var(--vh, 1vh) * 100);;
```

Comment:
same as previous comment

Reply:
The first height setting is a backup in case the browser the person is using doesn't support calc.

Reply:
ok. 2 semis on line 20

Reply:
thanks!